src/components/m-form-group/_template.js

```diff
@@ -11,19 +11,21 @@ export default function ({
 
   return [
     html`
-      <label class="m-form-group__wrapper js-form-group__wrapper"> 
-        <span class="m-form-group__label-icon-wrapper js-form-group__label-icon-wrapper">
-          <span class="m-form-group__label">
-            ${label}
+      <label class="m-form-group__wrapper js-form-group__wrapper">
+        <fieldset>
```

Comment:
`<fieldset>` within `<label>` is semantically and technically incorrect.
[`<label>`](https://www.w3.org/TR/2011/WD-html5-author-20110705/the-label-element.html) are only allowed to container [phrasing content](https://www.w3.org/TR/2011/WD-html5-author-20110705/content-models.html#phrasing-content).